techniques/general/G219.html

```diff
@@ -18,9 +18,9 @@ <h2>Applicability</h2>
 		</section>
 		<section id="description">
 			<h2>Description</h2>
-			<p>The objective of this Technique is to ensure that people with motor impairments who cannot carry out <a>dragging movements</a> are presented with an alternative <a>single pointer</a> interaction.</p> 
+			<p>The objective of this Technique is to ensure that people with motor impairments who cannot carry out <a>dragging movements</a> are presented with an alternative <a>single pointer</a> interaction that does not involve dragging.</p> 
```

Comment:
Was this single pointer mention intentionally preserved, or was it an oversight?